### DISCUSSION TITLE
Add optional environment variables at startup

### DISCUSSION THREAD

**Discussion by fliphess** [hassio-addons/addon-bitwarden#74] [Q&A] [answered]
Hi! I'd like to be able to change some environment variables at startup and I was wondering if said functionality would be appreciated when offered as a PR.

Preferably I add a list of env vars that can be set in the configuration as is done in (for example) the [grafana addon](https://github.com/hassio-addons/addon-grafana/blob/main/grafana/config.json#L16). 

When I add this feature to the addon, would you merge it or is it something I'd better use just in my own fork?

**Comment by frenck** [comment, marked as answer]
Those aren't needed, as the admin panel offers access to all features via a UI directly.

**Reply by fliphess** [reply]
Specifically, I'd like to change the `Database URL`, which is not available through the UI :)

**Reply by frenck** [reply]
Yet, I won't support that on the add-on either, as this messes with things like snapshots in ways people might not understand.

If you need this level of customization, this add-on is not for you.

**Reply by fliphess** [reply]
Cool 👍  I'll fork instead :-) I can image the support questions would drive you mad if you added that option :) 

Thanks for your quick response and have a good night :)

**Reply by frenck** [reply]
You too sir! 👍
